packages/apps/human-app/server/src/modules/auth-worker/auth-worker.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+import { ReputationOracleService } from '../../integrations/reputation-oracle.service';
+import {
+  SignupWorkerDto,
+  WorkerType,
+} from '../../interfaces/signup-worker-request.dto';
+
+@Injectable()
+export class AuthWorkerService {
+  constructor(private reputationOracleService: ReputationOracleService) {}
+
+  async signupWorker(signupWorkerDto: SignupWorkerDto) {
+    signupWorkerDto.type = WorkerType.WORKER;
+    return this.reputationOracleService.signupWorker(signupWorkerDto);
```

Comment:
service shouldnt operate on the Dto entity - make domain structure and mapper service

Reply:
Fixed in commit 8300eb3f